crates/cast/bin/tx.rs

```diff
@@ -401,3 +429,32 @@ where
         Ok(self)
     }
 }
+
+/// Helper function that tries to decode custom error name and inputs from error payload data.
+async fn decode_execution_revert(data: &RawValue) -> Result<Option<String>> {
+    if let Some(err_data) = data.to_string().replace("\"", "").strip_prefix("0x") {
```

Comment:
serde_json::from_value::<String is probably better

Reply:
changed to using `serde_json::from_str::<String>` in https://github.com/foundry-rs/foundry/pull/9794/commits/e9b5e66eb1af1ea3cc353da10427f12d7504ce94